samples/algorithms/database-search/host.py

```diff
@@ -0,0 +1,114 @@
+import math
```

Comment:
My apologies for forgetting to point this out in the previous round, but each new source file will need a header reminding people that the code is available under an MIT license. Thanks!

```suggestion
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.

import math
```